tests/test_handlers.py

```diff
@@ -722,45 +723,30 @@ def test_compute_or_load_am1_charges():
         np.testing.assert_array_equal(fresh, cached)
 
 
-@pytest.fixture
-def mol_with_precomputed_charges():
-    """Provide a test mol with partial charges precomputed on two different versions of Ubuntu"""
-    with resources.path("timemachine.testsystems.data", "ligands_40.sdf") as path_to_ligand:
-        mols = utils.read_sdf(path_to_ligand)
-    test_mol = mols[0]
+def assert_permutation_equivariance(mol, fxn, perm):
+    """fxn(mol[perm]) == fxn(mol)[perm]"""
+    mol = deepcopy(mol)
+    qs = fxn(mol)
 
-    # fmt: off
-    precomputed_charges = dict()
-    precomputed_charges['ubuntu-18.04.4'] = np.array([
-        -3.385959, 0.26308782, 2.30802986, 1.11718023, -2.11460365,
-        0.9725527, -9.18744615, 0.04042971, -2.49344069, 1.87709392,
-        -1.64877796, 0.57591713, -1.68873618, 1.19414994, -1.04834368,
-        -1.52407053, 0.30670004, -1.26003975, -1.3233364, 33.79971321,
-        -10.86357005, -10.86357005, -1.72362592, -0.10184044, -1.47951542,
-        -1.72362592, -11.32255943, 2.74250182, 1.64583123, 1.55000221,
-        1.69816584, 1.83477824, 2.11436794, 1.99956175, 1.98447416,
-        2.86426252, 2.86426252])
-    precomputed_charges['ubuntu-20.04.2'] = np.array([
-       -3.38772701, 0.26721331, 2.30272567, 1.11670881, -2.10741359,
-       0.96123709, -9.15420638, 0.03088217, -2.48495405, 1.872379,
-       -1.65467143, 0.58181069, -1.6896792, 1.19155677, -1.04775431,
-       -1.51994506, 0.3044605, -1.25933253, -1.32628322, 33.75586471,
-       -10.85178312, -10.85178312, -1.72386163, -0.10278342, -1.4785724,
-       -1.72386163, -11.31914145, 2.74450571, 1.64253075, 1.55165236,
-       1.69875511, 1.83418879, 2.1147215, 1.99413971, 1.98565289,
-       2.86638427, 2.86638427])
-    # fmt: on
+    permuted_mol = Chem.RenumberAtoms(mol, list(int(idx) for idx in perm))
+    qs_perm = fxn(permuted_mol)
+
+    np.testing.assert_allclose(qs_perm, qs[perm])
+
+
+@pytest.mark.parametrize("mol_idx", [0, 1, 2, 3, 4, 5])
+def test_partial_charge_equivariance_on_freesolv(mol_idx):
+    ff = Forcefield.load_default()
 
-    return dict(mol=test_mol, precomputed_charges=precomputed_charges)
+    seed = 2024
+    rng = np.random.default_rng(seed)
 
+    from timemachine.datasets import fetch_freesolv
```

Comment:
Move to the top of the file.  Unless we have a circular import here??

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/1397/commits/be9dda53d41ca9956f0882b106c37aea7be503ba